packages/api/cms-api/src/redirects/redirects.module.ts

```diff
@@ -18,14 +19,16 @@ export type RedirectsLinkBlock = OneOfBlock<
     CustomTargets & { internal: Block<InternalLinkBlockData, InternalLinkBlockInput>; external: typeof ExternalLinkBlock }
 >;
 
-interface Config {
+// TODO implement registerAsync instead of imports option?
+interface Config extends Pick<ModuleMetadata, "imports"> {
     customTargets?: CustomTargets;
     Scope?: Type<RedirectScopeInterface>;
+    targetUrlService?: Type<RedirectTargetUrlServiceInterface>;
```

Comment:
Shouldn't this be uppercase since it's a class (like Scope directly above)

Reply:
We're inconsistent here: for the ContentGenerationModule it's uppercase, for the UserPermissionsModule it's lowercase. We should harmonize that later. Which casing do you prefer?

Reply:
Discussed this and changed it to uppercase: https://github.com/vivid-planet/comet/pull/3624/commits/6cd7b4c66302d1e2e112a923320bc7e4b8ae3a07.